package.json

```diff
@@ -35,7 +35,6 @@
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^9.0.0",
     "@rollup/plugin-replace": "^2.3.3",
-    "babel-jest": "^24.9.0",
```

Comment:
We can remove `babel-jest` because `jest-config` always load `babel-jest` and we are not using `babel-jest` _explicitly_ in jest config.

https://github.com/facebook/jest/blob/e84a70c8456d61daa36b5c79a4df7ee752d6d6d6/packages/jest-config/src/normalize.ts#L241